content/blog/2019/06/2019-06-03-DevOps-World-Jenkins-World-2019-San-Francisco-Agenda-is-Live.adoc

```diff
@@ -0,0 +1,62 @@
+---
+:layout: post
+:title: "DevOps World-Jenkins World 2019 San Francisco: Agenda is Live"
+:tags:
+- event
+- devopsworldjenkinsworld2019
+author: alyssat
+---
+
+image::/images/post-images/dwjw-2019/2019-dwjw-email-san-fran-rev.png[2019 dwjw email san fran, role=center, float=left, height=256]
+
+We are a little over two months away from the largest Jenkins gathering of the year.  From Jenkins users, to maintainers, contributors, mentors and those new to Jenkins this event will have something for everyone.
+
+This year’s link:https://www.cloudbees.com/devops-world/san-francisco[**DevOps World - Jenkins World 2019 San Francisco**] has moved to a larger venue to facilitate the growth. From August 12 - 15, 2019 the event will take place at the Moscone West Center.  The event boasts 100+ sessions, and will offer training, hands-on workshops, onsite certification, contributor summit and much more.  Conference attendees can expect to be inspired while learning the latest innovations from industry leaders. Attendees will learn the value that digital transformation has in delivering software more efficiently, more quickly and with higher quality.
+
+We are excited to announce most of the agenda for DevOps World Jenkins World San Francisco is now live. We will continue to fill out the agenda with more sessions, trainings/workshops, and activities. Below is a small sampling of sessions from some of our favorite Jenkins contributors:
+
+=== Jenkins Configuration as Code: try it & start contributing! - link:https://github.com/ewelinawilkosz[Ewelina Wilkosz]
+
+link:https://github.com/jenkinsci/configuration-as-code-plugin[Jenkins Configuration as Code] is an open source Jenkins plugin that allows users to keep complete Jenkins configuration in a simple configuration file (yaml format). In the talk, I'll briefly present the history of the plugin, the vision for the future and current status. Then I'll move to the demo section where I'll show how easy it is to configure and run Jenkins with the help of the plugin.
+
+=== Thinking about Jenkins Security - link:https://github.com/MarkEWaite[Mark Waite]
```

Comment:
Wadeck is the real technical skills in this talk, let's be sure that he's mentioned as one of the presenters.

```suggestion
=== Thinking about Jenkins Security - link:https://github.com/MarkEWaite[Mark Waite & Wadeck Follonier]
```